src/main/java/riz/data/Task.java

```diff
@@ -0,0 +1,33 @@
+package riz.data;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public abstract class Task {
+    private String task;
+    private boolean completed;
```

Comment:
Perhaps, the boolean variable name can be more specific. For eg "isComplete".

Reply:
Yup thanks for the headsup!